configure.ac

```diff
@@ -4613,6 +4613,19 @@ AC_SUBST([OPT_CUPTI_IFLAGS])
 AC_SUBST([OPT_CUPTI_LDFLAGS])
 
 
+
+#-------------------------------------------------
+# Option: --with-opencl=PATH
+#-------------------------------------------------
+
+# Note: we will need location of intel/libOpenCL.so
+
+OPT_HAVE_OPENCL=yes
```

Comment:
set up OPENCL_IFLAGS

do we really need OPENCL_LDFLAGS? I don't think that we need to link against an OpenCL library.

Reply:
@jmellorcrummey , what are the different compliation flag types(other than LD and I flags)?

Reply:
done